WordPress/Classes/Networking/PeopleRemote.swift

```diff
@@ -13,21 +13,35 @@ class PeopleRemote: ServiceRemoteWordPressComREST {
         case UnknownError
     }
 
+
+    /// Specifies the number of entities to be retrieved on each query.
+    ///
+    private let pageSize = 50
+
+
     /// Retrieves the collection of users associated to a given Site.
     ///
     /// - Parameters:
     ///     - siteID: The target site's ID.
+    ///     - offset: The first N users to be skipped in the returned array.
     ///     - success: Closure to be executed on success
     ///     - failure: Closure to be executed on error.
     ///
     /// - Returns: An array of Users.
     ///
-    func getUsers(siteID: Int, success: ([User] -> Void), failure: (ErrorType -> Void)) {
+    func getUsers(siteID: Int,
+                  offset: Int = 0,
+                  success: ((users: [User], hasMore: Bool) -> Void),
+                  failure: (ErrorType -> Void))
+    {
         let endpoint = "sites/\(siteID)/users"
         let path = pathForEndpoint(endpoint, withVersion: .Version_1_1)
-        let parameters = [
-            "number": 50,
-            "fields": "ID, nice_name, first_name, last_name, name, avatar_URL, roles, is_super_admin, linked_user_ID",
+        let parameters: [String: AnyObject] = [
+            "number"    : pageSize,
+            "offset"    : offset,
+            "order_by"  : "display_name",
+            "order"     : "ASC",
+            "fields"    : "ID, nice_name, first_name, last_name, name, avatar_URL, roles, is_super_admin, linked_user_ID",
```

Comment:
I didn't opt to set the fields on Android, maybe I should have, but I was wondering if you're using `linked_user_ID` for anything.